handler.py

```diff
@@ -321,11 +321,12 @@ def main():
 
         try:
             if choice == '1':
-                item_id = int(input("Enter item ID: "))
-                name = input("Enter item name: ")
+                # Prompt the user to enter the item price and convert it to a float
                 price = float(input("Enter item price: "))
-                quantity = int(input("Enter item quantity: "))
-                store.add_item_to_inventory(item_id, name, price, quantity)
```

Comment:
change quantity to item quantity
```suggestion
                item_quantity = int(input("Enter item quantity: "))
                store.add_item_to_inventory(item_id, name, price, item_quantity)
```